test/ConfigTest.groovy

```diff
@@ -5,18 +5,30 @@ class ConfigTest {
   @Test
   public void defaultRuleSet() {
     def config = new Config([configFile: "/usr/src/app/fixtures/default/config.json", codeFolder: "/usr/src/app/fixtures/default"])
-    assertEquals config.ruleSet(), "/usr/src/app/ruleset.xml"
+    assertEquals "/usr/src/app/ruleset.xml", config.ruleSet()
   }
 
   @Test
   public void specifiedRuleSetFile() {
     def config = new Config([configFile: "/usr/src/app/fixtures/specified_file/config.new.json", codeFolder: "/usr/src/app/fixtures/specified_file"])
-    assertEquals config.ruleSet(), "/usr/src/app/fixtures/specified_file/rules.xml"
+    assertEquals "/usr/src/app/fixtures/specified_file/rules.xml", config.ruleSet()
   }
 
   @Test
   public void honorPresentRules() {
     def config = new Config([configFile: "/usr/src/app/fixtures/ruleset_default_file/config.json", codeFolder: "/usr/src/app/fixtures/ruleset_default_file"])
-    assertEquals config.ruleSet(), "/usr/src/app/fixtures/ruleset_default_file/ruleset.xml"
+    assertEquals "/usr/src/app/fixtures/ruleset_default_file/ruleset.xml", config.ruleSet()
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void doesNotAllowToMixRulesAndFile() {
+    def config = new Config([configFile: "/usr/src/app/fixtures/rules/config.mix.json", codeFolder: "/usr/src/app/fixtures/rules"])
+    config.ruleSet()
+  }
+
+  @Test
+  public void acceptRulesSimpleNames() {
+    def config = new Config([configFile: "/usr/src/app/fixtures/rules/config.json", codeFolder: "/usr/src/app/fixtures/rules"])
+    assertEquals "java-basic,java-design", config.ruleSet()
```

Comment:
Should this assertion be adjusted?

Reply:
That's the argument for the command line, should still be a single string.

Reply:
Oh, my mistake.

Should `config.mix.json` still contain a comma-separated string, though?

Reply:
Does not impact, but it should be adjusted. 👍